5.5/root/usr/bin/run-mysqld-slave

```diff
@@ -31,31 +25,34 @@ envsubst < ${CONTAINER_SCRIPTS_PATH}/my-paas.cnf.template > /etc/my.cnf.d/paas.c
 envsubst < ${CONTAINER_SCRIPTS_PATH}/my-slave.cnf.template > /etc/my.cnf.d/slave.cnf
 envsubst < ${CONTAINER_SCRIPTS_PATH}/my-tuning.cnf.template > /etc/my.cnf.d/tuning.cnf
 
-# Initialize MySQL database and wait for the MySQL master to accept
-# connections.
-initialize_database "$@"
-wait_for_mysql_master
+if [ ! -e "${MYSQL_DATADIR}/mysql" ]; then
```

Comment:
> If it's a restart, then the directory DOES exist.

isn't that what i said? 

i'm trying to confirm that when a slave restarts and the PV already contains old data, the slave will just pick up where it left off, rejoining as a slave w/ no need to go through any of the initialize_db/wait_for_master/etc steps that it went through when the PV was empty.